apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V58__fiss_claims_prov_fields.sql

```diff
@@ -0,0 +1,4 @@
+alter table "pre_adj"."FissClaims" add "provStateCd" varchar(2);
```

Comment:
we have a new V58 so will need to bump yours

Reply:
Version changed to V59 as requested.  We'll have to create a SQL only PR on Monday as well.